webapp/channels/src/components/user_group_popover/group_member_list/group_member_list.tsx

```diff
@@ -190,13 +191,16 @@ const GroupMemberList = (props: Props) => {
                             content={formatMessage({id: 'group_member_list.sendMessageTooltip', defaultMessage: 'Send message'})}
                         >
                             <DMButton
-                                className='btn-icon'
+                                className='btn btn-icon btn-xs'
                                 aria-label={formatMessage(
                                     {id: 'group_member_list.sendMessageButton', defaultMessage: 'Send message to {user}'},
                                     {user: name})}
                                 onClick={() => showDirectChannel(user)}
                             >
-                                <SendIcon/>
+                                <LocalizedIcon
+                                    className='icon icon-send'
+                                    ariaLabel={{id: t('user_group_popover.close'), defaultMessage: 'Close'}}
```

Comment:
In order to avoid the `t` function (something we want to get rid of) you should be able to use `defineMessage`.

I think it can be used here like this:
```suggestion
                                    ariaLabel={defineMessage({id: 'user_group_popover.close', defaultMessage: 'Close'})}
```

Nevertheless, I will leave the concrete directions on how to do this to @M-ZubairAhmed

Reply:
Best way would have been to avoid the use of LocalizeIcon altogether but i will not recommend that because its out of scope of this PR

Reply:
But i am confused didnt we wanted to move all icons towards compass icons it was recommended by the UX team as far as i remember

Reply:
@M-ZubairAhmed This is compass icon, it just adds an aria-label to the compass icon.